### ISSUE TITLE
Week 5 Error

### ISSUE THREAD

**Issue by vruel0529** [boilermaker-ben/Google-Sheets-NFL-Pick-Ems-and-Survivor-Pool#18]
Hi, for week 5 I'm trying to activate the code 'check NFL scores', but I keep receiving this message :

Exception: The number of rows in the data does not match the number of rows in the range. The data has 14 but the range has 16.

I imported all the responses for the survivor and I also updated the NFL schedule.

**Comment by vruel0529** [comment]
I think it has to do with the number of games this week because there are 14 games, but it is like we import 16 games.

**Comment by vruel0529** [comment]
Yes, I am doing the survivor only pool. So if I manually input the NFL_OUTCOMES everything will be fine? It worked last week.

**Comment by boilermaker-ben** [comment]
Yes, entering on the NFL_OUTCOMES should work as expected. Have you tried that for week 5? I haven't had anyone else message me about this error, so hopefully somehow its isolated to your pool somehow, but I do need to look into it still.

**Comment by trish11679-ux** [comment]
I just ran the check NFL scores and got the same issue for week 5
Exception: The number of rows in the data does not match the number of rows in the range. The data has 14 but the range has 16.

**Comment by boilermaker-ben** [comment]
I believe this is resolved now. Are you using a survivor-only pool?

Can you update to the [latest version of "apiPulling.gs"](https://github.com/boilermaker-ben/Google-Sheets-NFL-Pick-Ems-and-Survivor-Pool/blob/main/apiPulling.js) and try again?

**Comment by trish11679-ux** [comment]
Yes, I am only doing the survivor-only pool. I will update the apiPulling.gs now. - thank you if I have issues I will let you know.
